db.json

```diff
@@ -4,22 +4,26 @@
       {
         "id": 1,
         "name": "architecture",
-        "description": "Some archy description."
+        "description": "Some archy description.",
+        "tooltip": "{proportion} of users have the {x} architecture"
```

Comment:
Aren't template strings a built in? I just googled "Javascript template strings" and there's something that looks like `${proportion} of users have ${x} architecture` and wrapped in backticks?